docs/NewFeatures/lifecycle_expiration_policies.rst

```diff
@@ -0,0 +1,68 @@
+=============================
+
+Cloud users can apply lifecycle expiration rules (specified in Amazon’s
+`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
+to buckets managed through Zenko. These rules are triggered after a defined
+time has passed since the object’s creation. Zenko supports expiration of
+versioned or non-versioned objects, when a defined number of days has
+passed since those objects’ creation. This enables automatic deletion of
+older versions of versioned objects to reclaim storage space.

```

Comment:
Additionally, we can also expire current version of objects with separate rules, it's also supported in Orbit. (for versioned buckets, it adds a delete marker like when a user deletes the object without specifying a version ID).

Reply:
Using Zenko from the command line or from Orbit, you can expire the
current version of an object with a separate rule. For versioned buckets, add
a delete marker on an event, such as when a user deletes an object without first
specifying a version ID.